feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go

```diff
@@ -278,64 +280,64 @@ func testCiscoECNConfig(t *testing.T) {
 		desc:         "scheduler-policy-BE1",
 		sequence:     uint32(1),
 		priority:     oc.Scheduler_Priority_UNSET,
-		inputID:      "g_BE1",
+		inputID:      "BE1",
 		inputType:    oc.Input_InputType_QUEUE,
 		weight:       uint64(1),
-		queueName:    "g_BE1",
+		queueName:    "BE1",
```

Comment:
Are inputID and queueName always the same? If so, we can combine them?

Reply:
fixed